src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -2154,9 +2154,26 @@ private ISwitchExpressionOperation CreateBoundSwitchExpressionOperation(BoundSwi
         {
```

Comment:
Aleksey's comment didn't quite show up in a friendly place, he was indicating that we can change this method from accepting a `BoundSwitchExpression` to a `BoundConvertedSwitchExpression`.